### PR TITLE
Added S2S_URL

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1088

### Change description 
Flux: https://github.com/hmcts/cnp-flux-config/pull/2596

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/reform-scan-notification-service/values.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ java:
     DB_CONN_OPTIONS: ?sslmode=require
     FLYWAY_SKIP_MIGRATIONS: true
     QUEUE_NOTIFICATIONS_MAX_RETRY: 5
+    S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```
Comment (doncem) [inline charts/reform-scan-notification-service/values.yaml]: need to override the preview with aat value ;)

Comment (nitinprabhu) [reply charts/reform-scan-notification-service/values.yaml]: good catch

Comment (apereverzin) [reply charts/reform-scan-notification-service/values.yaml]: There is no S2S_URL in preview.template.yaml in processor

Comment (doncem) [reply charts/reform-scan-notification-service/values.yaml]: that's what we are saying - need to override (include into preview.values) to use aat value

**Pull request merged**